### PR TITLE
[SPARK-11773][SPARKR] Implement collection functions in SparkR.

### PR DESCRIPTION
**Pull request by sun-rui**

### CODE REVIEW

**Comment by shivaram** [comment]
LGTM. Thanks @sun-rui 

I checked some docs and it doesn't look like either of these verbs conflict with R's base package. Could you also double check ?

**Comment by felixcheung** [comment]
I don't think `array_contains` or `sort_array ` should mask anything in base or stats.

File: R/pkg/R/functions.R [hunk 1]
```diff
@@ -2365,3 +2333,74 @@ setMethod("rowNumber",
...
+            jc <- callJStatic("org.apache.spark.sql.functions", "size", x@jc)
+            column(jc)
+          })
+
+#' sort_array
+#'
+#' Sorts the input array for the given column in ascending order,
+#' according to the natural ordering of the array elements.
+#'
+#' @rdname sort_array
```
Comment (felixcheung) [inline R/pkg/R/functions.R]: could you add @param for `asc`?

**Comment by shivaram** [comment]
@felixcheung any other comments on this or is this good to merge ?

**Comment by felixcheung** [comment]
Looks good. Good to merge

**Pull request closed without merging**